lib/src/server.rs

```diff
@@ -491,69 +507,33 @@ impl Server {
     }
 
     pub fn run(&mut self) {
-        //FIXME: make those parameters configurable?
-        let mut events = Events::with_capacity(1024);
-        let poll_timeout = Some(Duration::milliseconds(1000));
-        let max_poll_errors = 10000;
-        let mut current_poll_errors = 0;
-        let mut last_zombie_check = Instant::now();
-        let mut last_sessions_len = self.sessions.borrow().slab.len();
-        let mut should_poll_at: Option<Instant> = None;
-        let mut last_shutting_down_message = None;
-
-        let mut loop_start = Instant::now();
-        loop {
-            let now = Instant::now();
-            time!("event_loop_time", (now - loop_start).whole_milliseconds());
-            loop_start = now;
+        let mut events = Events::with_capacity(1024); // TODO: make event capacity configurable?
+        self.last_sessions_len = self.sessions.borrow().slab.len();
 
-            if current_poll_errors == max_poll_errors {
-                error!(
-                    "Something is going very wrong. Last {} poll() calls failed, crashing..",
-                    current_poll_errors
-                );
-                panic!("poll() calls failed {} times in a row", current_poll_errors);
-            }
+        self.current_poll_errors = 0;
```

Comment:
To me, the `Server::run` function is called once, so these fields does not need to be assigned twice ;)

Reply:
sure!